src/apps/bristleback_apps/aanderaa/user_code/user_code.cpp

```diff
@@ -409,4 +416,117 @@ void loop(void) {
              current_data[i].getMin(), current_data[i].getMax());
     }
   }
+#endif // FAKE_AANDERAA
 }
+
+#ifdef FAKE_AANDERAA
+static void spoof_aanderaa() {
+  static constexpr uint32_t FAKE_STATS_PERIOD_MS = 60 * 1000;
+  static uint32_t fake_stats_timer_start = uptimeGetMs();
+  if (((u_int32_t)uptimeGetMs() - fake_stats_timer_start >= FAKE_STATS_PERIOD_MS)) {
+    SensorWatchdog::SensorWatchdogPet(AANDERAA_WATCHDOG_ID);
+    // Publish individual reading.
+    static AanderaaDataMsg::Data d;
+    size_t bufsize =
+        sizeof(payload_buffer); // Re-use the payload buffer since we don't need it anymore.
+    size_t encoded_len = 0;
+    d.header.version = AanderaaDataMsg::VERSION;
+    d.header.reading_uptime_millis = uptimeGetMs();
+    // Fill with random values in the range of 0-data from sensor log.
+    d.abs_speed_cm_s = rand() % 50;
+    d.abs_tilt_deg =  rand() % 7;
+    d.direction_deg_m = rand() % 337;
+    d.east_cm_s = rand() % -19;
+    d.heading_deg_m = rand() % 153;
+    d.north_cm_s = rand() % 41;
+    d.ping_count = rand() % 100;
+    d.tilt_x_deg = rand() % 2;
+    d.tilt_y_deg = rand() % 20;
+    d.max_tilt_deg = rand() % 3;
+    d.std_tilt_deg = rand() % 39;
+    d.temperature_deg_c = rand() % 20;
```

Comment:
Fine for now. We can continue tweaking later to be more realistic if it helps with testing. Just make sure it doesn't result in NAN for bad values.